pom.xml

```diff
@@ -1011,7 +1139,7 @@
         <maven.shadeplugin.version>2.4.3</maven.shadeplugin.version>
 
         <!-- dependency version properties -->
-        <tomcat.version>8.0.36</tomcat.version>
+        <tomcat.version>8.5.3</tomcat.version>
```

Comment:
There are some significant changes between tomcat 8.0 and 8.5. Please refer the migration guide [1] and verify whether the changes affect the extensions we have written.

[1] http://tomcat.apache.org/migration-85.html